app/profile/createTask/page.tsx

```diff
@@ -18,14 +18,14 @@ import ErrorMessage from "@/components/ErrorMessage/ErrorMessage";
 import Form from "@/components/Form/Form";
 
 export default function CreateTask() {
-  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [isLoading, setIsLoading] = useState(true);
   const [task, setTask] = useState<TaskType>({
     title: "",
     description: "",
     category: "default",
     importanceLevel: "default",
   });

```

Comment:
Nie musisz, ale warto na przyszłość wiedzieć: Do takich ciut większych struktur jak obiekty dobrym sposobem jest użycie useReducer zamiast useState. Możesz to po prostu zapamiętać i spróbować w przyszłości.